### PR TITLE
Add dao for quota

### PR DESCRIPTION
**Pull request by heww**
DAO for quota

Signed-off-by: He Weiwei <hweiwei@vmware.com>

### CODE REVIEW

File: src/common/models/quota.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	}
+
+	return hard, nil
+}
+
+// MustHard must returns quota hard
+func (q *Quota) MustHard() QuotaHard {
+	hard, err := q.GetHard()
+	if err != nil {
+		panic("hard invalid")
```
Comment (ywk253100) [inline src/common/models/quota.go]: Where panic here?

Comment (heww) [reply src/common/models/quota.go]: This method assert that `Hard` value in `Quota` instance is a valid `QuotaHard`.

Comment (ywk253100) [reply src/common/models/quota.go]: If the `Harbor` value is invalid, the whole process will exit, is it the expected behavior?

File: make/migrations/postgresql/0010_1.9.0_schema.up.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* add quota table */
+CREATE TABLE quota (
+ id SERIAL PRIMARY KEY NOT NULL,
+ reference VARCHAR(255),
```
Comment (wy65701436) [inline make/migrations/postgresql/0010_1.9.0_schema.up.sql]: not null?

File: make/migrations/postgresql/0010_1.9.0_schema.up.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ hard JSONB NOT NULL,
+ creation_time timestamp default CURRENT_TIMESTAMP,
+ update_time timestamp default CURRENT_TIMESTAMP,
+ UNIQUE(reference, reference_id)
+);
+
+/* add quota usage table */
+CREATE TABLE quota_usage (
+ id SERIAL PRIMARY KEY NOT NULL,
+ reference VARCHAR(255),
```
Comment (wy65701436) [inline make/migrations/postgresql/0010_1.9.0_schema.up.sql]: not null?

File: src/common/models/quota.go (hunk 1 above)
Comment (heww) [reply src/common/models/quota.go]: `MustHard` and `MustUsed` methods are removed, provide `equalHard` and `equalUsed` in suite instead.

File: make/migrations/postgresql/0010_1.9.0_schema.up.sql (hunk 2 above)
Comment (heww) [reply make/migrations/postgresql/0010_1.9.0_schema.up.sql]: Fixed

File: make/migrations/postgresql/0010_1.9.0_schema.up.sql (hunk 3 above)
Comment (heww) [reply make/migrations/postgresql/0010_1.9.0_schema.up.sql]: Fixed

**Pull request merged**